sdk/neonpostgres/arm-neonpostgres/CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # Release History
     
+## 2.0.0 (2025-05-14)
+    
+### Breaking Changes
```

Comment:
Could share the spec pr with me?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/34283

Reply:
Approved by Jeffrey. So accept this in JS side.